main.js

```diff
@@ -13,10 +13,19 @@ var timeErrMsg = document.querySelector('.time-err-msg');
 var studyBtn = document.getElementById('study');
 var meditateBtn = document.getElementById('meditate');
 var exerciseBtn = document.getElementById('exercise');
+var leftSection = document.querySelector('.left-section');
 var activityCards = [];
 var categorySelected;
+var categoryChoice2;
+var startTimerBtn = document.querySelector('#startTimerButton');
+var countdownText = document.querySelector('#countdownTimerText');
 
 submitForm.addEventListener("click", validate);
+startTimerBtn.addEventListener("click", beginCountDown);
+
+function beginCountDown() {
+  categoryChoice2.startTimer();
+}
 
 function switchLeftDisplay(){
    timerView.classList.toggle("hidden");
```

Comment:
We use classList.add on line 59.  I think we should use one or the other in the codebase.